src/games/strategy/triplea/ai/proAI/util/ProMatches.java

```diff
@@ -332,6 +332,21 @@ public boolean match(final Territory t) {
     };
   }
 
+  public static Match<Territory> territoryHasNonMobileInfraFactoryAndIsLand() {
```

Comment:
Pls move this method to be immediately below `territoryHasNonMobileInfraFactoryAndIsNotConqueredOwnedLand` and mark it as static. The request for the method move is to minimize vertical distance between first usage and method declaration.